index.js

```diff
@@ -2,11 +2,29 @@
 
 module.exports = cheapRuler;
 
+/**
+ * A collection of very fast approximations to common geodesic measurements. Useful for performance-sensitive code that measures things on a city scale.
+ *
+ * @name cheapRuler
```

Comment:
Are the `@name` tags necessary? Ideally both JSDoc and documentation.js should both be able to infer naming, and importantly name [turns off inference](http://usejsdoc.org/tags-name.html) in both engines, so mistakes like misnamed parameters won't be detected.